### ISSUE TITLE
report de algumas incompatibilidades 

### ISSUE THREAD

**Issue by rodrigopacheco100** [getomni/google-chrome-omni#12]
![image](https://user-images.githubusercontent.com/39929399/132894007-dc0d1960-fafe-4ea8-948e-27b4f1219fde.png)

No windows, a cor da fonte das abas que não estão abertas estão difíceis de ler e o botões de minimizar, maximizar e fechar estão com uma cor estranha.

**Comment by renato-ll** [comment]
Exato, e no Windows usando Brave fica igual com o mesmo problema, porem além disso os icones de fechar, maximizar e minimzar ficaram um pouco feios, acho que tinha que ser menor e menos chamativo:

![image](https://user-images.githubusercontent.com/38674705/153419224-16c376e6-2e02-44a0-b6bd-c394a1875e35.png)

**Comment by renato-ll** [comment]
Assim ficou bom;
![image](https://user-images.githubusercontent.com/38674705/153426766-8c6ad3db-a7d6-48a5-b846-050c3d3db247.png)

"theme_toolbar": "images/theme_toolbar.png"
![theme_toolbar](https://user-images.githubusercontent.com/38674705/153426942-e1a95ba1-822a-4c55-82c2-7d765e91f845.png)

resultado:
![image](https://user-images.githubusercontent.com/38674705/153427056-65aa4857-2392-44c9-90f7-5858989cf370.png)

**Comment by lucasmortoni** [comment]
eu fiz um fork desse repo e troquei a cor do background para abas inativas, posso fazer um pr, mas se quiserem customizar deem uma olhada lá. https://github.com/lucasmortoni/google-chrome.


cc @xinxilas @rodrigopacheco100 @jpedroschmitz
